modules/apps/user-associated-data/user-associated-data-api/src/main/java/com/liferay/user/associated/data/display/UADDisplay.java

```diff
@@ -53,16 +53,49 @@ public default String getEditURL(
 
 	public List<T> getRange(long userId, int start, int end);
 
+	/**
+	 * Returns an array of strings defining the field names to be used as table
+	 * column headers when sorting a list of entities of type {@code T}
+	 *
+	 * @return an array of field names used for column headers for sorting
+	 * @reivew
+	 */
 	public String[] getSortingFieldNames();
 
 	public String getTypeName(Locale locale);
 
+	/**
+	 * Returns an ordered range of all the entities of type {@code T} that match
+	 * the keywords with the given userId and groupIds.
```

Comment:
Ok so here's another one I had to fix from my original draft.  [Tip number 9 in our guidelines](https://github.com/liferay/liferay-portal/blob/master/readme/JAVADOC_GUIDELINES.markdown#top-tips) says: we should avoid referring to parameters by name, so in this case instead of `userId` and `groupIds`, it should be `user ID` and `group IDs`.